synfig-studio/src/gui/render.cpp

```diff
@@ -485,6 +485,7 @@ RenderSettings::submit_next_render_pass()
 
 		App::dock_info_->set_n_passes_pending(render_passes.size()); //! Decrease until 0
 		App::dock_info_->set_render_progress(0.0); //For this pass
+		App::dock_info_->enable_stop_button(true);
```

Comment:
Let the Dock_Info do its job by itself here.
Update the button sensitivity status inside the set_render_progress (if any value below 1.0, disable it) and set_async_render (if invalid, disable it)